examples/pytorch/README.md

```diff
@@ -0,0 +1,42 @@
+# PyTorch data loading with Mountpoint
+
+This directory shows a simple example of how to use Mountpoint for Amazon S3 as a PyTorch data loader via the torchdata library.
+It trains a ResNet-50 model for a few epochs using [PyTorch Lightning](https://www.pytorchlightning.ai/index.html), with synthetic ImageNet-style training data, stored in S3 as shards in WebDataset format.
```

Comment:
Would be great to have more info about the system you tested this workload, like os name or python version. Just by looking at the commands I would assume it's amazonlinux2023 and probably python2?

Reply:
I followed it using AL2023 and python3, and had to use `python3` instead of `python`. It would be good to be explicit in terms of "The following instructions were run using Amazon Linux 2023 and Python 3."

Should we consider making sure these instructions work out of the box for Python 3 (regardless of OS)?